### ISSUE TITLE
How do we set the icon in the taskbar on windows?

### ISSUE THREAD

**Issue by grofit** [tryphotino/photino.Blazor#61]
I have managed to set the window icon (although it looks a bit low quality for some reason), and the exe has an icon, but whenever its in the task bar it just shows up with a generic icon like so:

![image](https://user-images.githubusercontent.com/927201/193281730-5e83a98d-b371-4c1b-8dc0-af530ab5bb09.png)

Is there a specific outside of `SetIconFile` that we need to set?

**Comment by jairbubbles** [comment]
I believe it's the default icon you set in your executable. 

Just add a property `<ApplicationIcon>myIcon.icon</ApplicationIcon>` in your .csproj

**Comment by grofit** [comment]
I think the latest version has fixed it, as historically setting that didn't actually set the task bar icon.

**Comment by Kizari** [comment]
A note to anyone reading this in future. I ran into a similar issue despite setting the `<ApplicationIcon>` correctly in my `.csproj`. In my case, it seems Windows had cached a blank page icon against my application name (or something to that effect). I had already tried restarting my PC and publishing the application to a release folder to no avail. However, pinning the application to the taskbar and then unpinning it seemingly prompted Windows to refresh things and the icon now shows up. Give it a try if you're having the same problem!
